testdata/minio/minio.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: minio-deployment
+spec:
+  selector:
+    matchLabels:
+      app: minio
+  template:
+    metadata:
+      labels:
+        app: minio
+    spec:
+      initContainers:
+      - name: init-myservice
+        image: minio/minio:latest
+        command: ['mkdir', '-p', '/storage/test']
```

Comment:
Does this create the bucket instead of doing it in code?

Reply:
yes